### ISSUE TITLE
Judge cannot access RPE submissions if started scoring online

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#3958]
Program Team reported an issue with a judge who started online judging but was then assigned to an RPE. When that happened, they did not have the ability to view their RPE submissions. [Details are here](https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1682693216081869). I tried to recreate the issue on QA and ran into a similar but not exactly the same issue.

A judge clicked the button to start scoring online so there is an online submission in their "submissions to score" although they did not enter any data. 

<img width="1118" alt="Screen Shot 2023-04-28 at 12 11 00 PM" src="https://user-images.githubusercontent.com/516764/235199293-3f39a7fc-7fe8-4a02-9352-8d54c0cf2ba6.png">

They were then assigned to an RPE with specific submissions to judge. Those submissions appear in the submissions to score table too.

<img width="792" alt="Screen Shot 2023-04-28 at 12 22 16 PM" src="https://user-images.githubusercontent.com/516764/235201583-0f9807b5-6a33-4b73-9a3d-8e8bd461befb.png">

I clicked Start on "Coffee Defeats Orange Juice" and then clicked back to the submissions tab. Coffee Defeats Orange Juice is now listed twice - as online judging and a pitch event.

<img width="806" alt="Screen Shot 2023-04-28 at 12 24 17 PM" src="https://user-images.githubusercontent.com/516764/235201952-83b755ed-f39f-4e81-89ad-3faea67f948d.png">

If I actually fill out any of the score, the duplicate entry is removed and the score goes into the In Progress section. I was also able to remove myself from a submission.

<img width="829" alt="Screen Shot 2023-04-28 at 12 34 36 PM" src="https://user-images.githubusercontent.com/516764/235204082-7a02ce3f-23fa-41b4-9ff9-1f255b3915c2.png">

When I was looking at the issue Program Team reported, I did see the behavior where clicking on any RPE submission would only load the submission labeled for online judging, not the RPE submission. I did not have that same experience when testing on QA though.

**Comment by viviancan** [comment]
Added #3965 that captures the duplication issue

**Comment by viviancan** [comment]
Unable to recreate this bug locally. I also verified against prod data locally. With the fix for the duplication bug, the submissions are not duplicated in the submissions to score table. Locally, I deleted the one score they already started in the admin portal. I was able to go through the other RPE submissions without any issues. We can continue to monitor, but I am not able to verify if this was due to platform or user error.
